views/forms/DropdownCheckboxGroup/index.jsx

```diff
@@ -1,89 +1,119 @@
 import React, { useState } from 'react'
-import { useWatch } from 'react-hook-form'
-import { KeyboardArrowDown } from '@mui/icons-material'
-import { Menu, ListItem, TextField, InputAdornment } from '@mui/material'
-import ControlledCheckbox from '../ControlledCheckbox'
+import { FormControl, OutlinedInput, InputLabel, Select, MenuItem, Box, Chip } from '@mui/material'
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
```

Comment:
how confident are we that this will be the same across all instances of this component? Should we make it configurable?